<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="shubham-dhiman" failures="1" tests="2" name="cucumberOptions.TestRunner" time="5.382" errors="0" timestamp="2024-10-22T16:45:00 IST" skipped="0">
  <testcase classname="cucumberOptions.TestRunner" name="runScenario" time="0.211">
    <failure message="Forcefully failed expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Forcefully failed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at pageObjects.LoginPage.failedscenrios(LoginPage.java:74)
at stepDefinations.LoginPageStepDef.verify_failure(LoginPageStepDef.java:35)
at âœ½.verify failure(file:///C:/Users/shubham.dhiman/Desktop/Temp/JenkinsGithubDemo/src/test/java/Features/1.Login.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="cucumberOptions.TestRunner" name="runScenario" time="5.171"/>
  <system-out/>
</testsuite> <!-- cucumberOptions.TestRunner -->
